eng/config/PublishData.json

```diff
@@ -214,11 +214,19 @@
       "vsBranch": "rel/d16.9",
       "vsMajorVersion": 16
     },
-    "master-vs-deps": {
+    "release/dev16.10-vs-deps": {
       "nugetKind": ["Shipping", "NonShipping"],
       "version": "3.10.*",
       "packageFeeds": "default",
       "channels": [ "dev16.10", "dev16.10p1" ],
```

Comment:
```suggestion
      "channels": [ "dev16.10p1" ],
```

Reply:
what does this change do?

Reply:
Perhaps I misremember but I believe we let the most forward branch publish to the `dev16.x` channel and let preview branches continue to publish to the `dev16.xpy` channel.

Reply:
works for me!

Reply:
That being said, from looking at what the channels are updating (https://github.com/dotnet/versions/tree/master/build-info/dotnet/roslyn/), I am not sure we need to be maintaining channels anyway. I believe maestro and darc have surplanted this bit of infra.

Reply:
Yeah I'm not sure we need this anymore. Think thta was mostly useful for the manual nuget upload which we no longer do.